### PR TITLE
add mfojtik to owners for fileobserver

### PR DESCRIPTION
**Pull request by mfojtik**
/cc @sttts 
/cc @deads2k

### CODE REVIEW

**Comment by sttts** [comment]
lgtm

File: pkg/controller/fileobserver/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - deads2k
+  - sttts
+  - mfojtik
+approvers:
+  - smarterclayton
```
Comment (deads2k) [inline pkg/controller/fileobserver/OWNERS]: you can remove him

File: pkg/controller/fileobserver/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  - deads2k
```
Comment (deads2k) [inline pkg/controller/fileobserver/OWNERS]: don't need to duplicate top level

**Pull request merged**